Yosemite/Yosemite/Tools/Media/MediaType.swift

```diff
@@ -36,22 +34,18 @@ public enum MediaType {
         // The UT type identifier for both video/webm and audio/webm is "org.webmproject.webm"
         //
         // See https://github.com/woocommerce/woocommerce-ios/pull/4459/files#r656194065
-        guard "\(fileUTI)" != "org.webmproject.webm" else {
+        guard dataType != UTType("org.webmproject.webm") else {
             self = .other
             return
         }
 
-        if UTTypeConformsTo(fileUTI, kUTTypeImage) {
+        if dataType.conforms(to: UTType.image) {
             self = .image
-        } else if UTTypeConformsTo(fileUTI, kUTTypeVideo) {
+        } else if dataType.conforms(to: UTType.movie) {
             self = .video
-        } else if UTTypeConformsTo(fileUTI, kUTTypeMovie) {
-            self = .video
-        } else if UTTypeConformsTo(fileUTI, kUTTypeMPEG4) {
-            self = .video
```

Comment:
Video and MPEG4 types removed, because:
Video types conform to `video`, `video` [conforms](https://developer.apple.com/documentation/uniformtypeidentifiers/uttypevideo) to `movie`, `movie` is already in the list.

Reply:
Thanks for the explanation!